.jenkins/pipelines/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+                               make clean -C ${WORKSPACE}/solutions/sql_ae
+                               make -C ${WORKSPACE}/solutions/sql_ae
+                               make run -C ${WORKSPACE}/solutions/sql_ae
+                               """
+                    }
+                }
+            }
+        }
+        stage('Run Azure SDK tests') {
+            steps {
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                    withCredentials([string(credentialsId: 'Jenkins-ServicePrincipal-ID', variable: 'servicePrincipalId'),
+                                     string(credentialsId: 'ACC-Prod-Tenant-ID', variable: 'tenantId'),
+                                     string(credentialsId: 'Jenkins-ServicePrincipal-Password', variable: 'servicePrincipalKey'),
+                                     string(credentialsId: 'mystikos-ci-keyvault-url', variable: 'AZURE_KEYVAULT_URL'),
+                                     string(credentialsId: 'mystikos-ci-keyvault-url', variable: 'AZURE_TEST_KEYVAULT_URL'),
+                                     string(credentialsId: 'ACC-Prod-Subscription-ID', variable: 'AZURE_SUBSCRIPTION_ID'),
+                                     string(credentialsId: 'mystikos-storage-mystikosciacc-connectionstring', variable: 'STANDARD_STORAGE_CONNECTION_STRING')]) {
+                        sh """
+                           ${JENKINS_SCRIPTS}/global/run-azure-tests.sh \
+                             ${WORKSPACE}/tests/azure-sdk-for-cpp  \
+                             ${WORKSPACE}/solutions/dotnet_azure_sdk
+                           """
+                    }
+                }
+            }
+        }
+        stage('Run DotNet 5 Test Suite') {
+            steps {
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                    sh """
+                       make tests -C ${WORKSPACE}/solutions/coreclr
+                       """
+                }
+            }
+        }
+*/
+        stage('Measure Code Coverage') {
+            when {
+                expression { params.TEST_CONFIG == 'Code Coverage' }
+            }
+            steps {
+                sh """
+                   ${MYST_SCRIPTS}/myst_cc
+                   """
+            }
+        }
+        stage('Report Code Coverage') {
+            when {
+                expression { params.TEST_CONFIG == 'Code Coverage' }
+            }
+            steps {
+                sh """
+                   # Install .Net Code SDK and Cobertura
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/code-coverage/download-dotnet.sh
+
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   sudo apt install -y dotnet-sdk-5.0
+
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   sudo apt install -y cobertura
+
+                   # Publish Code Coverage Results
```

Comment:
Can publish to a storage blob. An example is: https://github.com/deislabs/mystikos/blob/main/.jenkins/release.Jenkinsfile#L199-L205